### ISSUE TITLE
Entities are not read correctly in specific DWG file

### ISSUE THREAD

**Issue by eugeniobidone** [DomCR/ACadSharp#141]
**Describe the bug**
In the attached DWG file the program finds only one geometric entity instead of the around 200 expected entities.

**To Reproduce**
Use the DwgReader.Read() function to read the attached DWG file obtaining the corresponding CadDocument object.

**Expected behavior**
The CadDocument object obtained from the attached DWG file should contain around 200 entities, not only one.

[silenziatore per oriano.zip](https://github.com/DomCR/ACadSharp/files/11718994/silenziatore.per.oriano.zip)
![Silenziatore per oriano](https://github.com/DomCR/ACadSharp/assets/136305718/ad50171b-b59f-458c-9d5b-bb98ef3722ae)

**Comment by DomCR** [comment]
Hi @eugeniobidone,

I've tested your file and I got 202 entities in the **ModelSpace**

Please check that you are using the latest Package in Nuget or update your branch with the latest changes, if the error persist please share the piece of code that you are using.

**Comment by eugeniobidone** [comment]
Hi @DomCR ,

I am using ACadSharp package in an Azure function, and when I run it in debug mode I obtain 1 entity in the ModelSpace. 

I have now tested the same file in a Console App, and I obtained 202 entities as expected.

Furthermore, the same Azure function after deployment reads 202 entities, so the problem appears only when I work in debug mode with my Azure function.

I don't know why it happens, but the important thing is that it works fine after deployment.

Thank you for your quick answer
